apollo-federation/src/supergraph/mod.rs

```diff
@@ -392,12 +395,14 @@ fn extract_subgraphs_from_fed_2_supergraph(
     Ok(())
 }
 
+#[allow(clippy::too_many_arguments)]
```

Comment:
don't love this, but I understand that it'll take a lot more time to refactor properly